### ISSUE TITLE
Let' decide main branch at which we work.

### ISSUE THREAD

**Issue by yuasawasabi** [nekokaeru/SE20#43]
In #39 the python program are converted to kotlin program.
How about working there.

**Comment by Johan-Berdougo** [comment]
I agree, people have started working on it, so we might as well go work on nanai-branch

**Comment by Leestalion** [comment]
I agree to work on the nanai-branch, it seems to implement every aspects of the problem.

**Comment by poyashi** [comment]
I agree, are there any remaining improvements?

**Comment by mt-revolution** [comment]
I agree because of its great computational speed and accuracy.
